### DISCUSSION TITLE
How to setBackgroundImage in v6 ?

### DISCUSSION THREAD

**Discussion by vaynevayne** [fabricjs/fabric.js#10217] [Q&A] [answered]
canvas.setBackgroundImage is deprecate
and  Image is deprecate 
How should I set up the background image now?
<img width="672" alt="image" src="https://github.com/user-attachments/assets/0b25663f-73db-4aad-9d35-b7d65dd04caa">

```jsx
import { Rect as BaseRect,Image } from 'fabric';


Canvas(id,{
backgroundImage:new Image('xxx')
})

```

**Comment by vaynevayne** [comment]
resolved   `import {  FabricImage } from 'fabric';`

**Comment by asturur** [comment, marked as answer]
Image and FabricImage are the same thing.
Image is the old name but it  conflicts with the standard api class Image from the Browser. So it has been renamed.
